environment.yml

```diff
@@ -16,6 +17,7 @@ dependencies:
   - rasterio>=1.0.16
   - shapely>=1.3.0
   - torchvision>=0.3
+  - torchaudio
```

Comment:
This should go before torchvision to be in alphabetical order

Reply:
Removed torchaudio dependency.